Dockerfile

```diff
@@ -46,7 +46,8 @@ RUN \
     /bin/bash scripts/install_narrative_docker.sh && \
     cd /tmp && \
     mkdir /tmp/narrative && \
-    chown -R nobody:www-data /tmp/narrative /kb/dev_container/narrative ; find / -xdev \( -perm -4000 \) -type f -print -exec rm {} \;
+    chown -R nobody:www-data /tmp/narrative /kb/dev_container/narrative && \
```

Comment:
With a semicolon, the `find` command would execute regardless of whether any of the previous steps errored out, so it was possible to create a narrative image where none of the JS or Python stuff was installed. 😱

Reply:
Oops.